pdf/invoice.tax-tab.tpl

```diff
@@ -48,7 +48,7 @@
         {assign var=label_printed value=false}
 
 			{foreach $bd as $line}
-				{if $line.rate == 0 and $label != 'ecotax_tax'}
+				{if $line.rate == 0}
```

Comment:
Actually, this one was needed because without this the eco-tax detail is never displayed when the ecotax rate is zero. But that's the only case where we need to display a tax which rate is zero.

Reply:
@jolelievre Could you check my change in `classes/pdf/HTMLTemplateInvoice.php` ? When I use the condition (with configuration), it set the line to 0 (which is remove in the loop just after).

Reply:
Yes I saw the other modifications, but if I get it right it means you rely on the value to be 0 to skip the display

So it works fine to prevent displaying it when the ecotax mode is disabled, however when ecotax is enabled and tax rate is zero it will also remove the line, which is what I tried to fix previously

Reply:
In this case, I think it should be managed in another PR the case of tax rate could be equals to 0. It should be decided by PM.

Reply:
I can create an issue if you want.

Reply:
Mmmm, since you are modifying the behaviour in this PR the question should be decided in this PR, not later. But I agree it needs to be validated with a PM.

Reply:
@PrestaShop/product-team What is the expected behavior when ecotax is enabled and tax rate is zero, do we display or not the line in invoice ?

Reply:
Let's be simple. We should always display the rate (no matter the rate) of the ecotax when the ecotax is enabled. 
It looks like a small improvement, does it realyy require an other PR ?